events/controller/config.go

```diff
@@ -0,0 +1,48 @@
+package controller
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/metal-toolbox/rivets/events"
+)
+
+const (
+	subjectPrefix = "com.hollow.sh.controllers.commands"
+)
+
+func queueConfig(appName, facilityCode, subjectSuffix, natsURL, credsFile string) events.NatsOptions {
+	// com.hollow.sh.controllers.commands.sandbox.servers.firmwareInstall
+	consumerSubject := fmt.Sprintf(
+		"%s.%s.servers.%s",
+		subjectPrefix,
+		facilityCode,
+		subjectSuffix,
+	)
+
+	return events.NatsOptions{
+		URL:            natsURL,
+		AppName:        appName,
+		CredsFile:      credsFile,
+		ConnectTimeout: time.Second * 60,
```

Comment:
just as easy to use `time.Minute` here instead of 60 * `time.Second` but also `gomnd` sucks and I wouldn't oppose getting rid of it.

Reply:
fixed, thanks!